src/com/elixr/training/Main.java

```diff
@@ -58,29 +65,14 @@ public static String readFile(String fileName) {
     }
 
     public static boolean isSupportedFile(File file) {
-        if (file.getName().endsWith(TXTFILE) || file.getName().endsWith(JSONFILE) ) {
+        if (file.getName().endsWith(Constants.TXTFILE) || file.getName().endsWith(Constants.JSONFILE)) {
             System.out.println("File format supported");
```

Comment:
Message move into constants class

Reply:
all error messages has been moved to constants class